compiler.js

```diff
@@ -339,6 +340,10 @@
                 else if (match.dec) {
                     section = this.addAttribute(match.dec[1] + '-=' + (match.dec[2] === undefined ? '1' : match.dec[2]), story, section, passage, isFirst, inputFilename, lineCount);
                 }
+                else if (match.deactivate) {
+                    section = this.addJS('$("[data-passage|=' + match.deactivate[1] + ']").addClass("disabled"); $("[data-passage|=' + match.deactivate[1] + ']").attr("tabindex", -1); squiffy.set("_turncount", squiffy.get("_turncount") + 1)', 
```

Comment:
I'd prefer this to be handled by calling a function in `squiffy.template.js`, rather than having JavaScript inside a string here.

The way I'd implement this would be to add a new data attribute on the generated link, then the `handleLink` function in `squiffy.template.js` does the deactivating. This also means everything that updates the turncount happens in the same function.